templates/custom/nodes/node--terrain--default.html.twig

```diff
@@ -0,0 +1,27 @@
+<h2>{{ label }}</h2>
+
+{% if content.field_park_offer|render|striptags|trim is not empty %}
+  <h3>{{ 'Park offer'|t }}</h3>
```

Comment:
klopt deze string? in het design staat "Ik wil naar een park" bij de facet.

Reply:
Das niet die string. Klopt wel denk ik. :)